src/metabase/driver/common.clj

```diff
@@ -432,7 +432,7 @@
            key
            class->base-type)))))
 
-(def ^:private days-of-week
+(def ^:private ^clojure.lang.PersistentVector days-of-week
```

Comment:
unrelated but by putting the tag here once we don't have to do it twice below

Reply:
I was using the first element in the array. Gets pretty brutal on sunday, the last element
```clojure
common=> (time
           (dotimes [_ 5000000]
             (.indexOf ^clojure.lang.PersistentVector days-of-week :sunday)))
"Elapsed time: 399.510708 msecs"
nil
common=> (let [days (zipmap days-of-week (range))]
           (time
             (dotimes [_ 5000000]
               (days :sunday))))
"Elapsed time: 43.934416 msecs"
nil
```

Reply:
If the difference is 10-360 milliseconds for 5 million calls it works out to 2-72 extra **microseconds** per 1 call which is hardly what I'd call brutal. I'm still 👍 on fixing it but I don't think it needs to be done as part of this PR especially since that code isn't part of this PR at all